lua/nvim-tree/git/runner.lua

```diff
@@ -82,9 +82,19 @@ function Runner:_run_git_job()
     pcall(uv.kill, pid)
   end
 
+  local function clean_log(opts_args)
```

Comment:
To clarify: are we just removing nils here?

We could just use a `vim.tbl_filter` if that's the objective.

Reply:
Yes, just removing the nils, and the suggestion is cleaner. I modified the function to use `vim.tbl_filter`.